pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.coveo</groupId>
     <artifactId>spillway</artifactId>
-    <version>2.1.2-SNAPSHOT</version>
+    <version>3.0.0-SNAPSHOT</version>
```

Comment:
so how is this breaking compared to the jedis 2.x version?

Reply:
It's breaking because some return types were changed in newest versions.

<img width="897" alt="Screen Shot 2022-04-20 at 3 37 32 PM" src="https://user-images.githubusercontent.com/12234108/164309396-9d0a93b4-98fc-4e36-9426-4b2aa6d0e562.png">

Reply:
yea but that's in the impl, in the spillway interface it's the same types returned

Reply:
yeah you're right, for the implementation it's not breaking.